utils/difference.js

```diff
@@ -0,0 +1,19 @@
+import transform from 'lodash.transform'
+import isEqual from 'lodash.isequal'
+import isObject from '@zap/utils/isObject'
+
+/**
+ * Deep diff between two object, using lodash
+ * @param  {Object} object Object compared
+ * @param  {Object} base   Object to compare with
+ * @return {Object}        Return a new object who represent the diff
+ */
+const difference = (object, base) => {
```

Comment:
Ideally, some tests would be great for this kind of utils, as they may be tricky sometimes

Reply:
Added a few basic tests for this.